### PR TITLE
Jake/leave group functionality

### PR DESCRIPTION
**Pull request by jakerogers2022**
## Trello Ticket :ticket: ##
https://trello.com/c/zTHl2oIw/72-be-fe-leave-group-functionality
## Summary :chart_with_upwards_trend: ##
Able to leave group from groupinfo page
## Test plan :clipboard: ##
working on android

### CODE REVIEW

File: screens/GroupInfo.js [hunk 1]
```diff
@@ -60,31 +61,54 @@ class GroupInfo extends Component {
...
 
+        //makes sure group hasn't been deleted 
+          if (firebaseDB.groups.hasOwnProperty(groupID) && firebaseDB.groups[groupID].groupMemberIds.hasOwnProperty(userId)) {
+              for (var user in firebaseDB.groups[groupID].groupMemberIds) {
+                  if (firebaseDB.users.hasOwnProperty(user)) {
+                      usersArray.push(firebaseDB.users[user].first_name + ' ' + firebaseDB.users[user].last_name);
+                  }
+              }
+              const moment = require('moment');
+              let today = moment().format('YYYY/MM/DD');
```
Comment (Peiyj) [inline screens/GroupInfo.js:80]: nit: you can change the type to const today = ....split..join

File: screens/GroupInfo.js [hunk 2]
```diff
@@ -60,31 +61,54 @@ class GroupInfo extends Component {
...
+    }
+    
+    leaveGroup = () => {
+        const groupID = this.state.groupID;
+        if (this.state.usernames.length == 1) {
+            const dbGroup = firebase.database().ref('/groups/' + groupID);
+            dbGroup.remove();
+        } else {
+            //TODO remove after dev
+            const userId = firebase.auth().currentUser ? firebase.auth().currentUser.uid : "testAdminId"
```
Comment (Peiyj) [inline screens/GroupInfo.js:105]: Would it make more sense to store userId as a state variable so that we don't repeatedly call to check?

**Review by Peiyj — APPROVED** [review]
LGTM! Some minor comments but overall great stuff!

File: screens/GroupInfo.js (hunk 2 above)
Comment (jakerogers2022) [reply screens/GroupInfo.js:105]: Yeah that's what I had originally, but I was getting some weird warnings so i switched it back.

**Pull request merged**